autorest/codegen/models/primitive_schemas.py

```diff
@@ -26,15 +26,13 @@ def _add_optional_and_default_value_template_representation(
     *,
     optional: bool = True,
     default_value_declaration: Optional[str] = None,
-    description: Optional[str] = None,
     **kwargs: Any
 ):
+    # for better display effect, use dafault value instead of var type
+    if default_value_declaration and default_value_declaration != "None":  # not doing None bc that's assumed
+        representation = default_value_declaration.strip("\"")
```

Comment:
I don't think we should get rid of the `\"` from the representation. Instead, what you can do is represent the json template with single quotes to get rid of this issue